### PR TITLE
updates: Update aws-sdk-rust to 1

### PR DESCRIPTION
**Pull request by ecpullen**
*Issue #, if available:*

*Description of changes:*
Updates the AWS SDK for rust to `1`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by webern** [comment]
Actions change needed

File: deny.toml [hunk 1]
```diff
@@ -23,22 +23,22 @@ allow = [
 ]
 
 exceptions = [
-    { name = "unicode-ident", version = "1.0.4", allow = ["MIT", "Apache-2.0", "Unicode-DFS-2016"] },
```
Comment (webern) [inline deny.toml]: Your IDE is making unnecessary formatting changes again.

File: Cargo.toml [hunk 2]
```diff
@@ -10,18 +10,16 @@ repository = "https://github.com/awslabs/coldsnap"
 keywords = ["AWS", "Amazon", "EBS", "snapshot"]
 
 [features]
-default = ["aws-sdk-rust-native-tls"]
-aws-sdk-rust-native-tls = ["aws-config/native-tls", "aws-sdk-ebs/native-tls", "aws-sdk-ec2/native-tls"]
-aws-sdk-rust-rustls = ["aws-config/rustls", "aws-sdk-ebs/rustls", "aws-sdk-ec2/rustls"]
+default = ["aws-sdk-rust-rustls"]
+aws-sdk-rust-rustls = [ "aws-config/rustls", "aws-sdk-ebs/rustls", "aws-sdk-ec2/rustls" ]
```
Comment (webern) [inline Cargo.toml:15]:
```suggestion
# This feature is for backward compatibility, but has no effect.
default = ["aws-sdk-rust-rustls"]
aws-sdk-rust-rustls = [ "aws-config/rustls", "aws-sdk-ebs/rustls", "aws-sdk-ec2/rustls" ]
```

**Review by webern — APPROVED** [review]

File: Cargo.toml (hunk 2 above)
Comment (webern) [reply Cargo.toml:15]: except my comma is unnecessary :roll_eyes:

**Review by jpculp — APPROVED** [review]
We're going to have to force merge this one, then remove the native-tls as a required check in our repo.

**Review by webern — APPROVED** [review]

**Review by jpculp — APPROVED** [review]
